### PR TITLE
fixed bug 984, deleted option Profile, setted up correct route for op…

### PR DESCRIPTION
**Pull request by skrebetsnataliia**
Deleted option "Profile" in the "User name drop-down list" (Header component).
Setted up correct route for option "Settings"
Previous:
bug
Result:
result

https://github.com/ita-social-projects/GreenCity/issues/984

### CODE REVIEW

**Comment by luchward** [comment]
Drop-down menu in the result contains two options, but its length is the same as for three options.  Would you fix it?

**Comment by skrebetsnataliia** [comment]
> Drop-down menu in the result contains two options, but its length is the same as for three options. Would you fix it?
Height of ul is defined by default. I fixed space between options. 
previous:
prev
results:
result

**Review by yulkats — APPROVED** [review]

File: src/app/component/layout/components/header/header.component.scss [hunk 1]
```diff
@@ -424,16 +424,15 @@ $screen-320: " screen and (max-width : 320px)";
...
+    display: inline;
     position: absolute;
     margin: 0;
     padding: 10px 0 0 0;
     background: #fff;
 
     li {
-      width: 100%;
-      height: 30px;
+      width: 141px;
```
Comment (mr-pavlenko) [inline src/app/component/layout/components/header/header.component.scss:434]: Why did you add the fixed width?

Comment (skrebetsnataliia) [reply src/app/component/layout/components/header/header.component.scss:434]: I added width according to mockup.
image

**Review by romanovskyroma — APPROVED** [review]

**Review by luchward — APPROVED** [review]

**Review by Benerous — APPROVED** [review]
